### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by jwilkins**
echo shows up in process history too

### CODE REVIEW

**Comment by MarcoFalke** [comment]
I think the goal was about not having the password in `[h]top` or `ps`.

If someone can `cat .bash_history`, they can likely do `cat mysecretsettings.conf` as well.

File: doc/release-notes.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The RPC command line client gained a new argument, `-stdin`
 to read extra arguments from standard input, one per line until EOF/Ctrl-D.
 For example:
 
-    $ echo -e "mysecretcode\n120" | src/bitcoin-cli -stdin walletpassphrase
+    $ cat mysecretsettings.conf | src/bitcoin-cli -stdin walletpassphrase
```
Comment (paveljanik) [inline doc/release-notes.md]: We will surely not merge `cat file | something reading from stdin...` ;-)

**Comment by paveljanik** [comment]
The `echo` command is used here as an example only. We should properly mark it as such, because it is in the shell history as well, of course. But not by using `cat` please...

**Comment by laanwj** [comment]
I did try this out before writing this, and echo won't show up in `ps` as it is a shell built-in, e.g. try this in one window:
```bash
$ echo "secret" | sleep 100
```
Then in another

```bash
ps eafux | grep secret
```

This shows only the grep process itself.

Sure, in practice, certainly in scripts you will probably source from a file
```bash
src/bitcoin-cli -stdin walletpassphrase < mysecretsettings.conf
```
In any case it just an example, you can use the functionality in different ways.

**Comment by paveljanik** [comment]
What about adding this after the example?
```
Warning: when used as described, the command line (including your secret
walletpassphrase) will be stored in your shell history file.
```

**Comment by laanwj** [comment]
Closing in favor of #8441

**Pull request closed without merging**